migrate_db.py

```diff
@@ -0,0 +1,26 @@
+# This script updates the database to the latest version.
+# It assumes the database has been initialized.
+from app import app
+from db import get_db
+
+def v1(db):
+    """ Update the database from v0 to v1 """
+    db.execute("ALTER TABLE member RENAME COLUMN receive_email TO subscribed")
+    return 1
+
+# List of functions that migrate the database.
+# They return the resulting version number.
+migrations = [
+    v1,
```

Comment:
Maybe make this a dict - so the versions they convert from are explicit?

Reply:
Sure, it makes no practical difference.